Makefile

```diff
@@ -143,9 +143,7 @@ install: default
 	$(INSTALL) -d /usr/local/apisix/
 	$(INSTALL) -d /usr/local/apisix/logs/
 	$(INSTALL) -d /usr/local/apisix/conf/cert
-	$(INSTALL) conf/mime.types /usr/local/apisix/conf/mime.types
-	$(INSTALL) conf/config.yaml /usr/local/apisix/conf/config.yaml
-	$(INSTALL) conf/config-default.yaml /usr/local/apisix/conf/config-default.yaml
+	$(INSTALL) conf/* /usr/local/apisix/conf/
```

Comment:
The `cert` dir inside `conf/` is not necessary in the production environment, I think it should be excluded.

Reply:
We need ssl ca placeholders to start APISIX. 
See #3013

Reply:
OK, got it.